File: src/main/java/project/chts/springboot/model/BasicScreening.java
package project.chts.springboot.model;

import javax.persistence.*;

@Entity
@Table(name="Basic_Screening")
public class BasicScreening {
	
	@Id
	private int child_id;
	
	@Column(name="head")
	private String head;
	
	@Column(name="neck")
	private String neck;
	
	@Column(name="skin_Color")
	private String skinColor;
	
	@Column(name="skin_Texture")
	private String skinTexture;
	
	@Column(name="mouth_Gums")
	private String mouthGums;
	
	@Column(name="mouth_Tongue")
	private String mouthTongue;
	
	@Column(name="hair_Color")
	private String hairColor;
	
	@Column(name="hair_Density")
	private String hairDensity;
	
	@Column(name="hair_Texture")
	private String hairTexture;
	
	@Column(name="chest")
	private String chest;
	
	@Column(name="abdomen")
	private String abdomen;
	
	@Column(name="treatment_For")
	private String TreatmentFor;
	
	@Column(name="referal")
	private String referal;
	
	@Column(name="reason_For_Referal")
	private String reasonForReferal;
	
	@Column(name="place_Referal")
	private String placeReferal;
	
	@Column(name="outcome")
	private String outcome;
	
	public BasicScreening() {
		
	}
	
	
	public BasicScreening(int child_id, String head, String neck, String skinColor, String skinTexture,
			String mouthGums, String mouthTongue, String hairColor, String hairDensity, String hairTexture,
			String chest, String abdomen, String treatmentFor, String referal, String reasonForReferal,
			String placeReferal, String outcome) {
		super();
		this.child_id = child_id;
		this.head = head;
		this.neck = neck;
		this.skinColor = skinColor;
		this.skinTexture = skinTexture;
		this.mouthGums = mouthGums;
		this.mouthTongue = mouthTongue;
		this.hairColor = hairColor;
		this.hairDensity = hairDensity;
		this.hairTexture = hairTexture;
		this.chest = chest;
		this.abdomen = abdomen;
		TreatmentFor = treatmentFor;
		this.referal = referal;
		this.reasonForReferal = reasonForReferal;
		this.placeReferal = placeReferal;
		this.outcome = outcome;
	}

	@Override
	public String toString() {
		return "BasicScreening [child_id=" + child_id + ", head=" + head + ", neck=" + neck + ", skinColor=" + skinColor
				+ ", skinTexture=" + skinTexture + ", mouthGums=" + mouthGums + ", mouthTongue=" + mouthTongue
				+ ", hairColor=" + hairColor + ", hairDensity=" + hairDensity + ", hairTexture=" + hairTexture
				+ ", chest=" + chest + ", abdomen=" + abdomen + ", TreatmentFor=" + TreatmentFor + ", referal="
				+ referal + ", reasonForReferal=" + reasonForReferal + ", placeReferal=" + placeReferal + ", outcome="
				+ outcome + "]";
	}

	public int getChild_id() {
		return child_id;
	}

	public void setChild_id(int child_id) {
		this.child_id = child_id;
	}

	public String getHead() {
		return head;
	}

	public void setHead(String head) {
		this.head = head;
	}

	public String getNeck() {
		return neck;
	}

	public void setNeck(String neck) {
		this.neck = neck;
	}

	public String getSkinColor() {
		return skinColor;
	}

	public void setSkinColor(String skinColor) {
		this.skinColor = skinColor;
	}

	public String getSkinTexture() {
		return skinTexture;
	}

	public void setSkinTexture(String skinTexture) {
		this.skinTexture = skinTexture;
	}

	public String getMouthGums() {
		return mouthGums;
	}

	public void setMouthGums(String mouthGums) {
		this.mouthGums = mouthGums;
	}

	public String getMouthTongue() {
		return mouthTongue;
	}

	public void setMouthTongue(String mouthTongue) {
		this.mouthTongue = mouthTongue;
	}

	public String getHairColor() {
		return hairColor;
	}

	public void setHairColor(String hairColor) {
		this.hairColor = hairColor;
	}

	public String getHairDensity() {
		return hairDensity;
	}

	public void setHairDensity(String hairDensity) {
		this.hairDensity = hairDensity;
	}

	public String getHairTexture() {
		return hairTexture;
	}

	public void setHairTexture(String hairTexture) {
		this.hairTexture = hairTexture;
	}

	public String getChest() {
		return chest;
	}

	public void setChest(String chest) {
		this.chest = chest;
	}

	public String getAbdomen() {
		return abdomen;
	}

	public void setAbdomen(String abdomen) {
		this.abdomen = abdomen;
	}

	public String getTreatmentFor() {
		return TreatmentFor;
	}

	public void setTreatmentFor(String treatmentFor) {
		TreatmentFor = treatmentFor;
	}

	public String getReferal() {
		return referal;
	}

	public void setReferal(String referal) {
		this.referal = referal;
	}

	public String getReasonForReferal() {
		return reasonForReferal;
	}

	public void setReasonForReferal(String reasonForReferal) {
		this.reasonForReferal = reasonForReferal;
	}

	public String getPlaceReferal() {
		return placeReferal;
	}

	public void setPlaceReferal(String placeReferal) {
		this.placeReferal = placeReferal;
	}

	public String getOutcome() {
		return outcome;
	}

	public void setOutcome(String outcome) {
		this.outcome = outcome;
	}


}

File: Components/BasicScreening.js
import './App.css';
import React, {useState, useEffect} from 'react';
import TemporaryDrawer from './navbar';
import TextField from '@mui/material/TextField';
import Paper from '@mui/material/Paper';
import {Container, Row, Col} from 'react-bootstrap';
import 'bootstrap/dist/css/bootstrap.min.css';
import ScreeningNav from './ScreeningNav';
import Box from '@mui/material/Box';
import Accordion from '@mui/material/Accordion';
import AccordionSummary from '@mui/material/AccordionSummary';
import AccordionDetails from '@mui/material/AccordionDetails';
import Typography from '@mui/material/Typography';
import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
import { useNavigate, useParams } from 'react-router-dom';

const BasicScreening = () => {

  const {child_id} = useParams();    
  const [head,  setHead] = useState('');
  const [neck,  setNeck] = useState('');
  const [skinColor,  setSkinColor] = useState('');
  const [skinTexture,  setSkinTexture] = useState('');
  const [mouthGums,  setMouthGums] = useState('');
  const [mouthTongue,  setMouthTongue] = useState('');
  const [hairColor,  setHairColor] = useState('');
  const [hairDensity,  setHairDensity] = useState('');
  const [hairTexture,  setHairTexture] = useState('');
  const [chest,  setChest] = useState('');
  const [abdomen,  setAbdomen] = useState('');
  const [treatmentFor,  setTreatmentFor] = useState('');
  const [referal,  setReferal] = useState('');
  const [reasonForReferal,  setReasonForReferal] = useState('');
  const [placeReferal,  setPlaceReferal] = useState('');
  const [outcome, setOutcome] = useState('');


  return (
    <div>
    
    <TemporaryDrawer/>
    <ScreeningNav child_id ={child_id}/>

    <div>
        <h4 className='page-screeninghead'>Basic Screening </h4>
        <p></p>
        <div>
          
          <Row>
          <Col sm={12}>

          <Paper className="form-add">
        <Box
       component="form"
         sx={{
        '& > :not(style)': { m: 1, width: '135ch' },
         }}
           noValidate
          autoComplete="off">
            <Accordion>
        <AccordionSummary
          expandIcon={<ExpandMoreIcon />}
          aria-controls="panel1a-content"
          id="panel1a-header"
        >
          <Typography>General Examination *</Typography>
          
        </AccordionSummary>
        <AccordionDetails>
          <Typography>
           
          
        <Box
       component="form"
         sx={{
        '& > :not(style)': { m: 1, width: '25ch' },
         }}
           noValidate
          autoComplete="off">
            <h4>General Examination *</h4>
          <TextField value ={head} name = "head" onChange={(e) => setHead(e.target.value)} label="Head/Scalp" variant="outlined" />
         <TextField value ={neck} name = "neck" onChange={(e) => setNeck(e.target.value)} label="Neck" variant="outlined" />
         <TextField value ={skinColor} name = "skinColor" onChange={(e) => setSkinColor(e.target.value)} label="Skin Colour" variant="outlined" />
         <TextField value ={skinTexture} name = "skinTexture" onChange={(e) => setSkinTexture(e.target.value)} label="Skin Texture" variant="outlined" />
         <TextField value ={mouthGums} name = "mouthGums" onChange={(e) => setMouthGums(e.target.value)} label="Mouth Gums" variant="outlined" />
         <TextField value ={mouthTongue} name = "mouthTongue" onChange={(e) => setMouthTongue(e.target.value)} label="Mouth Tongue" variant="outlined" />
         <TextField value ={hairColor} name = "hairColor" onChange={(e) => setHairColor(e.target.value)} label="Hair Colour" variant="outlined" />
         <TextField value ={hairDensity} name = "hairDensity" onChange={(e) => setHairDensity(e.target.value)} label="Hair Density" variant="outlined" />
         <TextField value ={hairTexture} name = "hairTexture" onChange={(e) => setHairTexture(e.target.value)} label="Hair Texture" variant="outlined" />
         <TextField value ={chest} name = "chest" onChange={(e) => setChest(e.target.value)} label="Chest" variant="outlined" />
         <TextField value ={abdomen} name = "abdomen" onChange={(e) => setAbdomen(e.target.value)} label="Abdomen" variant="outlined" />
         </Box>
         
          </Typography>
        </AccordionDetails>
      </Accordion>
    
      <Accordion>
        <AccordionSummary
          expandIcon={<ExpandMoreIcon />}
          aria-controls="panel1a-content"
          id="panel1a-header"
        >
          <Typography>Treatments *</Typography>
        </AccordionSummary>
        <AccordionDetails>
          <Typography>
         
        <Box
       component="form"
         sx={{
        '& > :not(style)': { m: 1, width: '25ch' },
         }}
           noValidate
          autoComplete="off">
          <h4>Treatments *</h4>
          <TextField value ={treatmentFor} name = "treatmentFor" onChange={(e) => setTreatmentFor(e.target.value)} label="Treatment for" variant="outlined" />
          <TextField value ={referal} name = "referal" onChange={(e) => setReferal(e.target.value)} label="Referal" variant="outlined" />
          <TextField value ={reasonForReferal} name = "reasonForReferal" onChange={(e) => setReasonForReferal(e.target.value)} label="Reason for Referal" variant="outlined" />
          <TextField value ={placeReferal} name = "placeReferal" onChange={(e) => setPlaceReferal(e.target.value)} label="Place Referal" variant="outlined" />
          <TextField value ={outcome} name = "outcome" onChange={(e) => setOutcome(e.target.value)}  label="Outcome" variant="outlined" />
         </Box>
         
          </Typography>
        </AccordionDetails>
      </Accordion>
         </Box>
         </Paper>

       <Paper className="form-add">
        <Box
       component="form"
         sx={{
        '& > :not(style)': { m: 1, width: '25ch' },
         }}
           noValidate
          autoComplete="off">
                 
         </Box>
         </Paper>
         </Col>
         </Row>

         <button type="submit" className="btn btn-primary save-button">Save</button>
         </div>
         </div>
    
    </div>
  );
}

export default BasicScreening;